src/js/pages/attendee/Layout.jsx

```diff
@@ -1,25 +1,151 @@
-import React from "react";
-import { Link } from "react-router";
+import React from 'react';
+import { Sidebar, SidebarItem } from 'react-responsive-sidebar';
+import Dialog from 'react-toolbox/lib/dialog';
 
-import Footer from "components/layout/Footer";
-import Nav from "components/layout/attendee/Nav";
+import Footer from 'components/layout/Footer';
+import BrandStore from 'stores/BrandStore';
+import ConferenceStore from 'stores/ConferenceStore';
+import PreferenceStore from 'stores/PreferenceStore';
+import AuthService from 'util/AuthService';
 
 export default class Layout extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      branding: {},
+      switcherModal: false,
+      newsfeed: PreferenceStore.getDefaults().newsfeed,
+      conferences: [],
+    };
+
+    BrandStore.getBranding();
+    PreferenceStore.loadAll();
+    ConferenceStore.getConferences();
+  }
+
+  componentWillMount() {
+    PreferenceStore.on('loaded', this.loadPreferences);
+    PreferenceStore.on('error', this.showError);
+    ConferenceStore.on('loadedConferences', this.loadConferences);
+    ConferenceStore.on('error', this.showError);
+    BrandStore.on('receivedBranding', this.setBranding);
+    BrandStore.on('error', this.showError);
+  }
+
+  componentWillUnmount() {
+    PreferenceStore.removeListener('loaded', this.loadPreferences);
+    PreferenceStore.removeListener('error', this.showError);
+    ConferenceStore.removeListener('loadedConferences', this.loadedConferences);
```

Comment:
gotta remove the listeners for the brand store as well. Memory links are no bueno